mlrun/api/utils/clients/iguazio.py

```diff
@@ -113,6 +126,40 @@ def verify_session(self, session: str) -> mlrun.api.schemas.AuthInfo:
         )
         return self._generate_auth_info_from_session_verification_response(response)
 
+    def get_or_create_access_key(
+        self, session: str, planes: typing.List[str] = None
```

Comment:
It is considered a bad practice in python to define default value that is of mutable type (list/dict/...) 
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
I do prefer to explicitly list data and control instead of using all (although now they are the same)

Reply:
So, is `SessionPlanes.all()` in use anywhere? While it may be useful someday to someone, for now it's just lying there... Not critical, but might be better to remove it.

Reply:
Not used, but I don't think it's harmful in any way